internal/api/dispatch/impl.go

```diff
@@ -82,6 +84,16 @@ func (this *dispatchManager) ProcessRun(ctx context.Context, account string, ser
 
 	entity := newRun(&run, correlationID, protocol.GetResponseFull(this.config), service, this.config)
 
+	if entity.OrgID == "" {
```

Comment:
As this is needed only for the one operation it would be better to do the translation in the controller (similarly to how the inverse translation is done for v2 of the operation).